testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -8719,6 +8719,10 @@ private static Consumer<List<? extends Throwable>> checkWarnings(
   }
 
   @Test void testJsonArray() {
+    // https://issues.apache.org/jira/browse/CALCITE-6003
```

Comment:
please add the jira url as a javadoc comment, like the other test do

Reply:
Done. Moved the new test to a separate function for that purpose, I have squashed the commits, since this is a small PR.